test/remote/gateways/remote_firstdata_e4_test.rb

```diff
@@ -136,10 +136,14 @@ def test_refund_with_track_data
     assert purchase = @gateway.purchase(@amount, @credit_card_with_track_data, @options)
     assert_match(/Transaction Normal/, purchase.message)
     assert_success purchase
-  
+
     assert response = @gateway.refund(50, purchase.authorization)
     assert_success response
     assert_match(/Transaction Normal/, response.message)
     assert response.authorization
   end
+
+  def test_dump_transcript
+    # dump_transcript_and_fail(@gateway, @amount, @credit_card, @options)
```

Comment:
Changed this to be `# See firstdata_e4_test.rb for an example of a scrubbed transcript`.